Reconstruction/PFA/Pandora/GaudiPandora/src/CaloHitCreator.cpp

```diff
@@ -447,7 +448,7 @@ pandora::StatusCode CaloHitCreator::CreateHCalCaloHits(const CollectionMaps& col
                     }
                     else{
                         Stave = m_decoder->get(pCaloHit->getCellID(), "stave");
-                        Stave = Stave <=2 ? Stave+5 : Stave-3 ;//FIXME , need check!!
+                        Stave = Stave ==0 ? Stave+7 : Stave-1 ;//correct, same with LCIO  
```

Comment:
It would be better to write some comments to explain these magic numbers.  Or you can add a few helper methods to hide these implementation. 
Tao

Reply:
Thanks for the graph. It is much clear.
Tao